.goreleaser.yaml

```diff
@@ -0,0 +1,115 @@
+version: 2
+
+before:
+  hooks:
+    - go mod tidy
+    - go run ./function-metadata/extract.go
+    - mv ./functions.txt ./lsp-server/com/embedded/
+    - gofmt -s -w .
+    - goimports -w .
+
+builds:
+  - env:
+      - CGO_ENABLED=0
+    dir: ./rad-build
+    binary: rad
+    goos:
+      - linux
+      - windows
+      - darwin
+      - freebsd
+    goarch:
+      - amd64
+      - arm
+      - arm64
+      - s390x
```

Comment:
nitpick but I wonder if we're going a bit overboard with these different targets atm? Should we just do amd64/arm and linux/windows/darwin (perhaps freebsd)?